corehq/util/datadog/__init__.py

```diff
@@ -0,0 +1,37 @@
+from django.conf import settings
+import logging
+
+logger = logging.getLogger(__name__)
+
+try:
+    from datadog.dogstatsd.base import DogStatsd
+    use_real_statsd = True
+except ImportError:
+    use_real_statsd = False
```

Comment:
Yea, this should only every happen locally since the datadog dependency is in prod-requirements.